app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The same here 🤔 Likes belongs-to user through the `'author_id'` column of the Like table. Now, we need to show that, the  `'author_id'` reference  User class modal.
Your code should look like this 👇🏻 :

```ruby
 # app/models/like.rb
  belongs_to :author, class_name: 'User', foreign_key: :author_id
```